src/axolotl/monkeypatch/accelerate/fsdp2.py

```
@@ -178,6 +178,38 @@ def get_state_dict(self, model, unwrap=True):
 
     return state_dict
 
+def cast_lora_module(module):
+    base_layer_dtype = module.base_layer.weight.dtype
+    # Linear4Bit will keep it's bias term in fp32. If the weight dtype is in bf16 we are not able to
+    # wrap this. Therefore we must ensure the bias has the same dtype as the weight
+    if hasattr(module.base_layer, "bias") and module.base_layer.bias is not None:
+        if module.base_layer.weight.dtype != module.base_layer.bias.dtype:
+            log_bias_dtype_mismatch = True
```

Ruff (0.12.2)

Local variable `log_bias_dtype_mismatch` is assigned to but never used

Remove assignment to unused variable `log_bias_dtype_mismatch`

(F841)